### ISSUE TITLE
Slight artefacts when using black and white colors

### ISSUE THREAD

**Issue by mheesakkers** [micycle1/PeasyGradients#11]
When testing different types of gradients I noticed that especially at the radial gradient that some black turns up in a place where it should be white. It is also slightly noticeable with the conic variant (it appears as a small line on the left). I tried a couple of interpolation and colorspace modes but got no different results.

```
gradient = new Gradient(
      color(0, 0, 100),
      color(0, 0, 0)
    );
gradient.setColorSpace(ColorSpaces.DIN99);
gradient.setInterpolationMode(Interpolation.LINEAR);
```

![Screenshot 2022-09-29 at 20 40 55](https://user-images.githubusercontent.com/5702106/193115976-b553aa7d-9c84-4d5c-aceb-9a4d7a76462b.png)

Also when using color the artefact comes back.
![Screenshot 2022-09-29 at 20 47 56](https://user-images.githubusercontent.com/5702106/193117214-5585c87c-138f-4c9c-b7d3-0ae96b5a92ad.png)

**Comment by micycle1** [comment]
My initial guess would be this is identical to #9. Do other color spaces exhibit the same issue?

**Comment by mheesakkers** [comment]
Works with [the latest build](https://github.com/micycle1/PeasyGradients/actions/runs/3154877856)! Thanks!
![Screenshot 2022-09-30 at 07 50 31](https://user-images.githubusercontent.com/5702106/193199432-df61d86a-cf91-40c0-9be9-706b1903aa14.png)

**Comment by stigmollerhansen** [comment]
Happy to learn the bug is fixed in the latest build (0.9.3 at the current time of writing). However, I can't find the .jar file? Any chance you can provide this? I haven't got the Java skills to compile source code myself. Massive props for writing and providing this library. It's awesome!

**Comment by micycle1** [comment]
> Happy to learn the bug is fixed in the latest build (0.9.3 at the current time of writing). However, I can't find the .jar file? Any chance you can provide this? I haven't got the Java skills to compile source code myself. Massive props for writing and providing this library. It's awesome!

I'll upload the fat-jar soon!
